### PR TITLE
Still not fixed not sure why I bother anymore haha

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
_**FOURTH**_ **TIME MAKING THIS PR HAHA**
_Now with 85% less pasta!_
Adds Makeshift Switchblades, essentially glorified Butcher's Cleavers that can retract.

## Why It's Good For The Game
More variety when it comes to petty crime, weapons crafting, self defense tools, and antaging! Gives you a reason to use all those modular recievers for something other than your 80th makeshift shotgun this round. Then again, it might just be easier to autolathe a Butcher's Cleaver proper.

## Changelog
:cl:
add: Makeshift Switchblades, a weaker subtype of normal switchblades that can be crafted using a Kitchen Knife, Modular Receiver, Rifle Stock and some Cable Coil. Requires a welder to complete.
/:cl:
_"You're a big guy, ain't ya?_

### CODE REVIEW

**Comment by ShadeAware** [comment]
Special thanks to **@monkey king#6510** on Discord for helping me fix this pile of bullshit I call a PR.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -337,6 +337,32 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+	icon_state = "switchblade_ms"
+	desc = "A concealable spring-loaded knife."
+	force = 2
+	throwforce = 3
+	
+/obj/item/switchblade/makeshift/attack_self(mob/user)
+	extended = !extended
+	playsound(src.loc, 'sound/weapons/batonextend.ogg', 50, 1)
+	if(extended)
+		force = 15
```
Comment (Ghommie) [inline code/game/objects/items/weaponry.dm:350]: Make three new variables for the blade force, throwforce and icon_state to be set to when extended on `switchblade/attack_self` (values should be 20, 23 and "switchblade_ext"), then set such variables to 15, 15 and "switchblade_ext_ms" on this subtype.
None of this janky pasta please.
See [transforming.dm](https://github.com/Citadel-Station-13/Citadel-Station-13/blob/master/code/game/objects/items/melee/transforming.dm) for reference.

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -337,6 +337,32 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+		w_class = WEIGHT_CLASS_NORMAL
+		throwforce = 15
+		icon_state = "switchblade_ext_ms"
+		attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+		hitsound = 'sound/weapons/bladeslice.ogg'
+		sharpness = IS_SHARP
+	else
+		force = 1
```
Comment (Ghommie) [inline code/game/objects/items/weaponry.dm]: Now, you'll have to change the equivalent line on `switchblade/attack_self` to 
```
		force = initial(force)
```
lest the retracted makeshift switchblade force reset to the base non-makeshift switchblade instead.

File: code/game/objects/items/weaponry.dm [hunk 3]
```diff
@@ -337,6 +337,32 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+		w_class = WEIGHT_CLASS_SMALL
+		throwforce = 2
```
Comment (Ghommie) [inline code/game/objects/items/weaponry.dm]:
```
		throwforce = initial(throwforce)
```

File: code/game/objects/items/weaponry.dm [hunk 4]
```diff
@@ -337,6 +337,32 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+		icon_state = "switchblade_ms"
```
Comment (Ghommie) [inline code/game/objects/items/weaponry.dm]:
```
		icon_state = initial(icon_state)
```

**Review by Ghommie — CHANGES_REQUESTED** [review]
Nothing in the sprites suggests it's a **makeshift** switchblade, specially when compared to that funny recipe that's more fit for a janky knife launcher or a gunblade.
In the current state it's just a slighty stronger knife with no redeeming feature, not sure how it should be merged.

**Comment by ShadeAware** [comment]
Remind me to not make PRs when im half awake
Ill try and fix these Soon™

**Comment by ShadeAware** [comment]
> 
> 
> Nothing in the sprites suggests it's a **makeshift** switchblade, specially when compared to that funny recipe that's more fit for a janky knife launcher or a gunblade.
> In the current state it's just a slighty stronger knife with no redeeming feature, not sure how it should be merged.

I liked the original sprite and really didn't want to change it but it needed some differences from the Gangs one so I just recolored the grip to look like wood

**Comment by Ghommie** [comment]
Well, that's just a switchblade with a wooden grip. :smirk:

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (ShadeAware) [reply code/game/objects/items/weaponry.dm:350]: Forgive me, but from what I can see the blade force, throwforce is already set 15, and the icon is set correctly to switchblade_ext. Unless im missing something. Im probably missing something.

**Comment by ShadeAware** [comment]
Now that I think about it a name change may be in order but im not sure to what yet :thinking:

**Comment by ShadeAware** [comment]
> Nothing in the sprites suggests it's a **makeshift** switchblade, specially when compared to that funny recipe that's more fit for a janky knife launcher or a gunblade.
> In the current state it's just a slighty stronger knife with no redeeming feature, not sure how it should be merged.

And to elaborate as to why I think this is a good idea, it's mainly for a sense of realistic consistency when it comes to self defense tools. For example, not everyone is going to be carrying around a big kitchen knife on their person, a butcher's cleaver even less so. A switchblade on the other hand is much more likely, as it is a small yet robust and easily concealed weapon. Additionally, to say that it's just a stronger knife with no additional benefits as a reason to why it cannot be considered is flawed, because you could say that very same thing about the butcher's cleaver; it's just a more robust knife.

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weaponry.dm:350]: this is the switchblade overhaul i was talking about

**Comment by ShadeAware** [comment]
wait fuck, thats not supposed to happen

**Comment by ShadeAware** [comment]
REEEE WRONG BRANCH ITS NOT SUPPOSED TO TOUCH MY EYES

**Comment by ShadeAware** [comment]
### **WHY DO I EVEN FUCKING BOTHER**

**Comment by ShadeAware** [comment]
**LET ME FIX THIS FOR A FOURTH FUCKING TIME**

**Comment by ShadeAware** [comment]
GOD FUCKING DAMN IT

**Pull request closed without merging**